### PR TITLE
Added support of the steiger config for windows

### PR DESCRIPTION
**Pull request by mx-ryabov**
Hey! 
My version of tests looks a bit overwhelming, I also thought about a simple function that would take a platform parameter and depending on the value would prepend `C`:, but I'm not sure if that's the right way to do tests. However, if you think otherwise, I can rework it (or am open to any other suggestions)

### CODE REVIEW

File: .changeset/early-stingrays-appear.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'steiger': minor
+---
+
+Added support of the steiger config for windows
```
Comment (illright) [inline .changeset/early-stingrays-appear.md:5]: suggestion: the support is already there, it's just buggy :)


```suggestion
Fix drive label handling in paths on Windows 
```

File: .changeset/early-stingrays-appear.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'steiger': minor
```
Comment (illright) [inline .changeset/early-stingrays-appear.md:2]: Also I think it should be a patch change, since it's a bug fix:
```suggestion
'steiger': patch 
```

**Review by illright — CHANGES_REQUESTED** [review]
Thanks for making a PR! Sorry it took a while to get a review. The solution looks solid. The tests are a bit hard to parse, I think we can make them simpler by simply writing two `it`s and then skipping one or the other based on the platform. Our CI runs in both OSes, so we will cover everything that way

For reference: https://vitest.dev/api/#test-runif

**Comment by mx-ryabov** [comment]
I considered a similar to yours approach but I thought about the case when somebody on a unix machine could break something for windows and this developer won't find out it until the CI checks passed.
But anyway all your comments are resolved, let me know if you need something else. Thank you!

**Review by illright — APPROVED** [review]
Thanks for addressing the comments! Nothing else on my side, great job, let's merge.

As for your concern about devs not finding out until CI — there's a lot of places where that can happen, and that's okay, that's why we have CI on several OSes, to alert devs that something might not work on someone else's machine
